src/test/java/dev/openfeature/sdk/FlagEvaluationSpecTest.java

```diff
@@ -135,10 +135,13 @@ public void initialize(EvaluationContext evaluationContext) throws Exception {
         assertEquals(h2, api.getHooks().get(1));
     }
 
-    @Specification(number="1.1.6", text="The API MUST provide a function for creating a client which accepts the following options:  - name (optional): A logical string identifier for the client.")
-    @Test void namedClient() {
-        assertThatCode(() -> api.getClient("Sir Calls-a-lot")).doesNotThrowAnyException();
-        // TODO: Doesn't say that you can *get* the client name.. which seems useful?
+    @Specification(number="1.1.6", text="The API MUST provide a function for creating a client which accepts the following options:  - domain (optional): A logical string identifier for binding clients to provider.")
+    @Test void domainName() {
+        assertNull(api.getClient().getMetadata().getDomain());
```

Comment:
+1 for addressing the todo

Reply:
no problem, was a good opportunity to resolve it!